src/main/java/org/c4sg/dao/OrganizationDAO.java

```diff
@@ -24,5 +30,13 @@
     @Query(FIND_BY_NAME_OR_DESCRIPTION)
     List<Organization> findByNameOrDescription(@Param("name") String name, @Param("description") String description);
 
+    
+    @Transactional
+    @Modifying
+    @Query(UPDATE_PROJECT_UPDATED_TIME)
+    Integer updateProjectUpdatedTime(@Param("projectUpdatedTime") Date projectUpdatedTime, @Param("orgId") Integer orgId);
```

Comment:
@gauravhp try this... `Organization localOrgan = localProject.getOrganization();
            localOrgan.setProjectUpdatedTime(new Date(Calendar.getInstance().getTime().getTime()));
            organizationDAO.save(localOrgan);`